plugins/woocommerce/includes/import/class-wc-product-csv-importer.php

```diff
@@ -696,6 +696,12 @@ public function parse_download_file_field( $value ) {
 	 * @return int
```

Comment:
We should consider changing the return type, as it will also return an empty string from now on.

Reply:
Good catch! Updated!